humanmade/amf-wordpress
# feat(local-multisite): enable skipping API requests

#### Background

In a multisite setup, making an API request feels like a waste of resources. This opt-in functionality would enable a user to skip making an actual API Request, and instead running a query for the correct blog id. 


#### Test instructions
 - Add the const before the plugin is loaded
 ```
   define("AMF_LOCAL_BLOG_ID", 1);
 ```
- Set a breakpoint, and observe an API request is not made, but the results are still the same.
